src/components/pages/ServiceMarketplace/index.tsx

```diff
@@ -63,39 +68,62 @@ export default function ServiceMarketplace() {
   const [selected, setSelected] = useState<string>('All Services')
   const [sortOption, setSortOption] = useState<string>('new')
   const [cardServices, setCardServices] = useState<ServiceRequest[]>([])
-
-  let serviceTypeId = ''
-
-  if (selected === ServiceTypeIdsEnum.DATASPACE_SERVICES) {
-    serviceTypeId = ServiceTypeIdsEnum.DATASPACE_SERVICE
-  } else if (selected === ServiceTypeIdsEnum.CONSULTANCY_SERVICES) {
-    serviceTypeId = ServiceTypeIdsEnum.CONSULTANCY_SERVICE
-  }
-
-  let sortingType = 'ReleaseDateDesc'
-  if (sortOption === 'provider') {
-    sortingType = 'ProviderDesc'
-  }
-
-  const indexToSplit = 2 //show only 2 services in recommended
-
-  const { data, error, isError, refetch } = useFetchServicesQuery({
+  const [page, setPage] = useState<number>(0)
+  const [serviceTypeId, setServiceTypeId] = useState<ServiceTypeIdsEnum | ''>(
+    ''
+  )
+  const [sortingType, setSortingType] = useState<SORTING_TYPE>(
+    SORTING_TYPE.RELEASE_DATE_DESC
+  )
+  const [argsData, setArgsData] = useState<{
+    page: number
+    serviceType: ServiceTypeIdsEnum | ''
```

Comment:
better use undefined instead of the empty string
`serviceType?: ServiceTypeIdsEnum`